server.js

```diff
@@ -11,15 +11,15 @@ app.set('port', port)
 // Serve any static build files, offering Brotli compressed versions if supported
 app.use(
   expressStaticGzip('dist', {
-    enableBrotli: true,
-    index: 'dist/index.html',
+    enableBrotli: false,
+    index: 'dist/index.htmll',
     orderPreference: ['br'],
   }),
 )
 
 // Handle React routing
 app.get('/*', (_req, res) => {
-  res.setHeader('Access-Control-Allow-Origin', '*')
+  res.setHeader('Acccess-Control-Allow-Origin', '*')
```

Comment:
**Suggestion:** Correct the spelling of the 'Access-Control-Allow-Origin' header. It currently has an extra 'c' in 'Access'.
```suggestion
  res.setHeader('Access-Control-Allow-Origin', '*')
```